### ISSUE TITLE
onion site connection

### ISSUE THREAD

**Issue by piermark** [radicallyopensecurity/netaidkit-legacy#92]
From browser is possibile navigate into onion site?

**Comment by agent725** [comment]
Hi piermark,

Are you sure your NetAidKit is connected to TOR, and your computer is connected to the NetAidKit? In that case you should be able to visit .onion domains without a problem.

In some cases it helps to reconnect your computer to the NetAidKit, as it happens that computers do not always update their DNS IP's.

Regards,
Agent725

**Comment by piermark** [comment]
Yes, also with netaidkit trunk version, .Onion site working very well with Chrome, Explorer and Firefox on Windows, on Mac  only with Chrome working, on iPhone/iPad don't work
